### DISCUSSION TITLE
How to map the escape key?

### DISCUSSION THREAD

**Discussion by FranklinYu** [gdh1995/vimium-c#1135] [Q&A]
The “Custom Key Mappings” section mentioned some special keys like `<tab>` and `<enter>`, but not the escape key. I tried

```viml
map <esc> zoomReset
```

but nothing happens when I press the Escape key. The same command works if I map it to another key (such as <kbd>r</kbd>). Is the escape key supported? If so, what is the correct name for it?

**Comment by gdh1995** [comment]
No you can not map `<esc>` to a command. If you indeed want to do so, you need `mapKey`:
```
mapKey <esc> <fake_key>
map <fake_key> zoomReset
```
